index.bs

```diff
@@ -96,32 +96,32 @@ interface CompressionStream {
 CompressionStream includes GenericTransformStream;
 </pre>
 
-The {{CompressionStream}}(format) constructor, when invoked, must run these steps:
-    1. If <dfn for=CompressionStream>format</dfn> is unsupported in CompressionStream, then throw a TypeError.
+The {{CompressionStream}} (format) constructor, when invoked, must run these steps:
+    1. If <dfn for=CompressionStream>format</dfn> is unsupported in CompressionStream, then throw a [TypeError](https://www.ecma-international.org/ecma-262/6.0/#sec-native-error-types-used-in-this-standard-typeerror).
     1. Let *cs* be a new CompressionStream object.
     1. Set *cs*'s *format* to <a for=CompressionStream>format</a>.
     1. Let *startAlgorithm* be an algorithm that takes no arguments and returns nothing.
     1. Let *transformAlgorithm* be an algorithm which takes a *chunk* argument and runs the <a>compress and enqueue a chunk</a> algorithm with *cs* and *chunk*.
     1. Let *flushAlgorithm* be an algorithm which takes no argument and runs the <a>compress flush and enqueue</a> algorithm with *cs*.
-    1. Let *transform* be the result of calling <a abstract-op>CreateTransformStream</a>(*startAlgorithm*, *transformAlgorithm*, *flushAlgorithm*).
+    1. Let *transform* be the result of calling <a abstract-op>CreateTransformStream</a> (*startAlgorithm*, *transformAlgorithm*, *flushAlgorithm*).
```

Comment:
Are you sure we should have a space before `()`? The other standards I work on ([Streams](https://streams.spec.whatwg.org/), [Encoding](https://encoding.spec.whatwg.org/) and [Fetch](https://fetch.spec.whatwg.org/) do not put a space before the parameters.